backend/app/tests/services/collections/test_delete_collection.py

```
@@ -0,0 +1,217 @@
+from sqlmodel import Session
+from sqlalchemy.exc import SQLAlchemyError
+
+
+from app.models.collection import (
+    DeletionRequest,
+    Collection,
+    ResponsePayload,
+)
+from app.tests.utils.utils import get_project
+from app.crud import CollectionCrud
+from app.crud import CollectionCrud, CollectionJobCrud
```

Ruff (0.13.1)

Redefinition of unused `CollectionCrud` from line 15

Remove definition: `CollectionCrud`

(F811)